File: src/main/java/com/graqr/threshr/model/redsky/products/InStoreOnly.java
package com.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record InStoreOnly(
        @JsonProperty("availability_status")
        String availabilityStatus) {
}

File: src/main/kotlin/com/graqr/threshr/model/redsky/product/InStoreOnly.kt
package com.graqr.threshr.model.redsky.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class InStoreOnly(
    @param:JsonProperty("availability_status") val availabilityStatus: String
) {}
